agnpy/synchrotron/synchrotron.py

```diff
@@ -90,18 +90,13 @@ def __init__(self, blob, ssa=False, integrator=np.trapz):
         self.ssa = ssa
         self.integrator = integrator
 
-    def electron_energy_loss_per_time(self, gamma):
-        # For synchrotron, energy loss dE/dt formula is:
-        # (4/3 * sigma_T * c * magn_energy_dens) * gamma^2
-        # In case of constant B, the part in brackets is fixed, so as an improvement we could calculate it once
-        # and cache, and later we could use the formula (fixed * gamma^2)
-        fixed = self._electron_energy_loss_formula_prefix()
+    def electron_energy_loss_rate(self, gamma):
+        fixed = self._electron_energy_loss_formula_prefactor()
         return fixed * gamma ** 2
 
-    def _electron_energy_loss_formula_prefix(self):
-        # using SI units here because of the ambiguity of the different CGS systems in terms of mu0 definition
-        magn_energy_dens = (self.blob.B.to("T") ** 2) / (2 * mu0)
-        return ((4 / 3) * sigma_T * c * magn_energy_dens).to("erg/s")
+    def _electron_energy_loss_formula_prefactor(self):
```

Comment:
not sure if it makes sense to split it into two functions, especially that you do not do it for IC losses

Reply:
It's split, because the "prefactor"  calculation is done in some test, so I didn't want to duplicate the formulate, and instead extracted it to the (sub-)function for reuse in those tests